tests/integration/test_s3.py

```diff
@@ -789,6 +789,14 @@ def test_s3_request_payer(self):
         self.assertEqual(response['Payer'], 'Requester')
         self._delete_bucket(bucket_name)
 
+    def delete_non_existing_bucket(self):
+        bucket_name = 'test-%s' % short_uid()
+        try:
```

Comment:
Thanks for this PR @macnev2013 👍 

nit: Better use `self.assertRaises` here:
```
with self.assertRaises(ClientError) as ctx:
    self.s3_client.delete_bucket(Bucket=bucket_name)
self.assertEqual(ctx.exception.response['Error']['Code'], 'NoSuchBucket')
self.assertEqual(ctx.exception.response['Error']['Message'], 'The specified bucket does not exist')
```

Reply:
Updated.